uahf-technical-spec.md

```diff
@@ -199,6 +199,13 @@ digest based on BIP143. Revisions were made to account for non-Segwit
 deployment.
 
 
+### REQ-7 Difficulty adjustement in case of hashrate drop
+
+In case the MTP of the tip of the chain is 12h after the MTP 6 block before,
```

Comment:
This is imprecise.

It should be "12 hours _or more_ after ..." 

> the MTP 6 block before

This is also imprecise. Better: 

> the MTP of the chain 6 blocks before the tip